docs/_renderer.py

```diff
@@ -21,6 +21,12 @@ def render(self, el: qd.ast.ExampleCode) -> str:
         expect_failure = "quartodoc: +EXPECTED_FAILURE"
         quartodoc_skip_doctest = "quartodoc: +SKIP"
 
+        # Avoid needing this boilerplate in every example and docstring
+        prelude_lines = [
+            ">>> import ibis",
+            ">>> ibis.options.interactive = True",
```

Comment:
Some of our examples aren't interactive, and it might be weird to see

```
>>> import ibis
>>> ibis.options.interactive = True
>>> ibis.options.interactive = False
```

in those examples.

Another awkward thing is having `import ibis` at the beginning of **every** code example.

For example

```python
"""Docstring for function.

Examples
--------
Do some stuff

>>> import ibis
>>> ibis.options.interactive = True

Continue doing stuff based on previous stuff

>>> import ibis
>>> ibis.options.interactive = True  # confusing why this is here again
"""
```

Reply:
Yes, that seems unworkable.

What if we put this prelude stuff in a separate code block, with a directive to quarto to hide the whole thing.

Reply:
I'm not sure what we get out of that. While it _is_ boilerplate, I'm not sure it's worth the hidden code that's being executed on the developer's behalf.

Reply:
Roger. I'm fine closing this out then!

Reply:
Thanks! Keep the PRs coming!